core.js

```diff
@@ -76,10 +77,11 @@ export function clearPackageJSON(packageJson, inputIgnoreFields) {
   const ignoreFields = inputIgnoreFields
     ? IGNORE_FIELDS.concat(inputIgnoreFields)
     : IGNORE_FIELDS
-  const cleanPackageJSON = filterObjectByKey(
-    applyPublishConfig(packageJson),
-    key => !ignoreFields.includes(key) && key !== 'scripts'
-  )
+
+  const cleanPackageJSON = structuredClone(applyPublishConfig(packageJson))
+  // Delete ignore fields from packageJson except 'scripts'
+  ignoreFields.forEach(field => field !== 'scripts' && deleteProperty(cleanPackageJSON, field))
```

Comment:
Adding 2 dependencies just to delete key is not a good strategy for code maintainability and avoiding supply chain attacks.

JS community should stop adding dependencies for simple tasks.